
Fix config for /icons/ dir to allow symlink to poweredby.png.

--- httpd-2.4.2/docs/conf/extra/httpd-autoindex.conf.in.iconlink
+++ httpd-2.4.2/docs/conf/extra/httpd-autoindex.conf.in
@@ -21,7 +21,7 @@ IndexOptions FancyIndexing HTMLTable Ver
 Alias /icons/ "@exp_iconsdir@/"
 
 <Directory "@exp_iconsdir@">
-    Options Indexes MultiViews
+    Options Indexes MultiViews FollowSymlinks
     AllowOverride None
     Require all granted
 </Directory>
